lib/charms/mongodb/v1/shards_interface.py

```diff
@@ -325,6 +354,44 @@ def get_related_shards(self) -> List[str]:
         """Returns a list of related shards."""
         return [rel.app.name for rel in self.charm.model.relations[self.relation_name]]
 
+    def get_unreachable_shards(self) -> List[str]:
+        """Returns a list of unreable shard hosts."""
+        if not self.model.relations[self.relation_name]:
+            logger.info("shards are not reachable, none related to config-sever")
+            return False
```

Comment:
why we are returning `False` if  return type List[str] ? 😄

Reply:
covered in Mykolas comment